editoast/openapi.yaml

```diff
@@ -4086,7 +4086,33 @@ components:
               values:
                 description: List of `n+1` values associated to the ranges
                 items:
-                  type: string
+                  oneOf:
+                  - properties:
+                      Electrification:
+                        properties:
+                          voltage:
+                            type: string
+                        required:
+                        - voltage
+                        type: object
+                    required:
+                    - Electrification
+                    type: object
+                  - properties:
+                      NeutralSection:
+                        properties:
+                          lower_pantograph:
+                            type: boolean
+                        required:
+                        - lower_pantograph
+                        type: object
+                    required:
+                    - NeutralSection
+                    type: object
+                  - description: Not electrified section
+                    enum:
+                    - NotElectrified
```

Comment:
I believe we use `NonElectrified` everywhere else ([in editoast](https://github.com/OpenRailAssociation/osrd/blob/bde48441f61f96ad2be1d658dcbe2591c789b272/editoast/src/models/train_schedule.rs#L451) and [in core](https://github.com/OpenRailAssociation/osrd/blob/bde48441f61f96ad2be1d658dcbe2591c789b272/core/envelope-sim/src/main/java/fr/sncf/osrd/envelope_sim/electrification/NonElectrified.java#L4)).

Reply:
Fixed